content/tests/components/recipes/details/RecipeDetails.test.js

```diff
@@ -89,6 +91,16 @@ describe("The `RecipeDetailForm` component", () => {
     return recipe;
   };
 
+  const experimenterSetup = (experiment) => {
+    if (!experiment) {
+      experiment = experimenterResponseFactory.build();
+    }
```

Comment:
It's a little more verbose, but since JS parameter defaults are evaluated at calltime, you can do

```suggestion
  const experimenterSetup = (experiment = experimenterResponseFactory.build()) => {
```